generators/server/templates/src/main/kotlin/package/security/oauth2/AuthorizationHeaderFilter.kt.ejs

```diff
@@ -38,9 +38,7 @@ class AuthorizationHeaderFilter(private val headerUtil: AuthorizationHeaderUtil)
     override fun run(): Any? {
         val ctx = RequestContext.getCurrentContext()
         val authorizationHeader = headerUtil.getAuthorizationHeader()
-        authorizationHeader?.run {
-            ctx.addZuulRequestHeader(TokenRelayRequestInterceptor.AUTHORIZATION, this)
-        }
+        authorizationHeader.ifPresent { ctx.addZuulRequestHeader(TokenRelayRequestInterceptor.AUTHORIZATION, it) };
```

Comment:
This could be avoided if return type of AuthorizationHeaderUtil.getAuthorizationHeader() is changed to String? instead of Optional<String>. See also comment in AuthorizationHeaderUtil.